src/lib/utils/urls.js

```diff
@@ -12,10 +12,13 @@ export default {
       login: "/api/user/login",
       logout: "/api/user/logout",
       getCurrent: "/api/user",
+      getStandards: "/api/user/standards/get/",
+      updateStandards: "/api/user/standards/update",
```

Comment:
It'd be slightly more intuitive to have these URLs be accessed with `api.user.standards.get` and `api.user.standards.update`. Very minor things here, but this discrepancy _could_ be why someone in the future has a bad day haha